lib/publiccloud/instance.pm

```diff
@@ -23,6 +23,7 @@ use db_utils;
 use Mojo::Util 'trim';
 use Data::Dumper;
 use mmapi qw(get_current_job_id);
+use publiccloud::provider qw(provider_client);
```

Comment:
`provider_client` is not a method exported by `provider` 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/provider.pm

Reply:
yes I updated it